webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
@adam-be But as you just mentioned, "update connection's ICE gathering/connection state" are queued. That means they already happen *after* functions passed to `.then` execute, because promises operate on the microtask queue, whereas "queue a task" refers to the main event loop.